amundsen_gremlin/utils/streams.py

```diff
@@ -354,6 +354,7 @@ def _consumer(things: Iterable[V], ignored: None) -> None:
     return _actual_state
 
 
+# NB: This will not work on python 3.6; requires 3.7 or later
```

Comment:
is this the only functionality that won't work on 3.6?

Reply:
Yep! asyncio changed significantly between 3.6 and 3.7. 

Fortunately we don't use this in the proxy, and folks can use the synchronous one for their own databuilder if they cannot upgrade to 3.7